Assets/Mirage/Runtime/NetworkBehaviour.cs

```diff
@@ -147,6 +148,20 @@ protected internal void SetSyncVarHookGuard(ulong dirtyBit, bool value)
         /// </summary>
         NetworkIdentity _identity;
 
+
+#if !UNITY_2020_1_OR_NEWER
```

Comment:
Considering 2019.4 LTS is soon to be unsupported (2022 Q1?), I believe this part can be removed as it adds some complexity

Reply:
Shall I remove all pre-2020 compatibility parts and commit again?

Reply:
This is my opinion, @dragonslaya84 what do you think?

Reply:
> This is my opinion, @dragonslaya84 what do you think?

Ya i agree. We are only supporting lts and up. And 2019 is about to be unsupported so might as well obsoleteit now

Reply:
> Shall I remove all pre-2020 compatibility parts and commit again?

Please do remove it :)

Reply:
> Please do remove it :)

Done! ;)

Reply:
Thanks!

Reply:
I am still using 2019.4 in one of my projects so I'll still need it to work there for a few months.

I think we should be able to just keep the old code for 2019 and just have the fixed version for 2020+. I'll remove the old 2019 code after unity stops supporting it (maybe add a `// todo` so we dont forget)

Reply:
Reverted the commit, added TODO.